### PR TITLE
[iOS] adjusments to platform view frame and scrollview initialization.

### PR DESCRIPTION
**Pull request by jonahwilliams**
If the platform view bounds haven't changed, we don't need to update its frame. this is a small but measurable performance improvement on the text views platform view example. This also simplifies the code: rather than changing the frame and then untransforming the origin. We can just let the origin be 0, 0. Seems to work!

image

### CODE REVIEW

**Comment by jonahwilliams** [comment]
Up next: maybe its time to remove the mask pool so we can avoid reinitializing them every frame?

File: shell/platform/darwin/ios/framework/Source/FlutterViewController.mm [hunk 1]
```diff
@@ -1386,12 +1386,15 @@ - (void)viewDidLayoutSubviews {
   CGRect viewBounds = self.view.bounds;
   CGFloat scale = [self flutterScreenIfViewLoaded].scale;
 
-  // Purposefully place this not visible.
```
Comment (jonahwilliams) [inline shell/platform/darwin/ios/framework/Source/FlutterViewController.mm:1389]: Somewhat unrelated but this also shows up in profiles

**Comment by hellohuanlin** [comment]
> Up next: maybe its time to remove the mask pool so we can avoid reinitializing them every frame?

I was actually thinking about it when trying to use CALayer (rather than UIView) as the mask.

**Comment by jonahwilliams** [comment]
Looks like this doesn work :(

**Pull request closed without merging**